sdcm/utils/es_queries.py

```diff
@@ -35,12 +37,16 @@ def filter_test_details(self):
         test_details += ' AND test_details.test_name: {}'.format(self.test_name.replace(":", r"\:"))
         return test_details
 
+    def filter_test_for_lastyear(self):
+        year_ago = (datetime.today() - timedelta(days=365)).date().strftime("%Y%m%d")
```

Comment:
what will happen with tests (from previous versions) that we ran in the same year? the results will be filtered out? why?
shouldn't this be an option?

Reply:
The query will search for all test results for all versions for last year. i think it is not critical, because scylla has 2 release per year. But it could be implemented as options.